migrate.py

```diff
@@ -1198,6 +1146,57 @@ def assemble_collections(checkout_path, spec, args, target_github_org):
             REMOVE = set()
 
 
+def galaxy_metadata_init(collection, namespace, target_github_org):
+    return {
+        'namespace': namespace,
+        'name': collection,
+        'version': '1.0.0',  # TODO: add to spec, args?
+        'readme': None,
+        'authors': None,
+        'description': None,
+        'license': None,
+        'license_file': None,
+        'tags': None,
+        'dependencies': {},
+        'repository': f'git@github.com:{target_github_org}/{namespace}.{collection}.git',
+        'documentation': None,
+        'homepage': None,
+        'issues': None
+    }
+
+
+def process_symlink(namespace, collection, plugin_type, dest, src, spec):
+    real_src = os.readlink(src)
+
+    # remove destination if it already exists
+    if os.path.exists(dest):
+        # NOTE: not atomic but should not matter in our script
+        logger.warning('Removed "%s" as it is target for symlink of "%s"' % (dest, src))
```

Comment:
The best practice is to let the logger interpolate vars internally. It's usually flagged by pylint as `logging-format-interpolation` warning: http://pylint.pycqa.org/en/latest/technical_reference/features.html#logging-checker-messages.

```suggestion
        logger.warning('Removed "%s" as it is target for symlink of "%s"', dest, src)
```

Another reason is that it helps the advanced logging configs to track individual var values.

Reply:
Going to fix this in the whole file.

Reply:
Okay, then it deserves a separate PR.